### PR TITLE
Space class#8

### PR DESCRIPTION
**Pull request by reenz**
Added space class
Added loop in ERB to display multiple spaces

### CODE REVIEW

**Comment by AlexandraGF** [comment]
🐰

**Comment by bpourian** [comment]
🦑

File: app/app.rb [hunk 1]
```diff
@@ -51,10 +54,15 @@ def current_user
...
-    session[:space_price] = params[:space_price]
-    session[:space_availability] = params[:space_availability]
+    Space.create(space_name:params[:space_name],
+      space_description: params[:space_description],
+      space_price: params[:space_price],
+      availability_start: params[:availability_start],
+      availability_end: params[:availability_end],
+      user: current_user)
+
+      p params[:space_name]
```
Comment (samuelmaker) [inline app/app.rb]: can be removed?

**Comment by samuelmaker** [comment]
💪🏽

**Pull request merged**